src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/FlagsEnumTests/EnumParseStyleFlagsTests.cs

```diff
@@ -0,0 +1,198 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text.Json.Serialization;
+using Xunit;
+using static System.AttributeTargets;
+
+namespace System.Text.Json.Tests.Serialization.FlagsEnumTests
```

Comment:
Please fold the two new test files you created into https://github.com/dotnet/runtime/blob/main/src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/EnumConverterTests.cs.